plugins/doc_fragments/dimensiondata.py

```diff
@@ -19,7 +19,6 @@ class ModuleDocFragment(object):
   region:
     description:
       - The target region.
-    choices:
```

Comment:
That's the most interesting `choices` values I've ever seen. I had to check out how it looks in the docs: https://docs.ansible.com/ansible/2.10/collections/community/general/dimensiondata_network_module.html#parameter-region